rayon-core/src/lib.rs

```diff
@@ -144,7 +144,7 @@ impl Configuration {
         if self.num_threads > 0 {
             self.num_threads
         } else {
-            match env::var("RAYON_RS_NUM_CPUS").ok().and_then(|s| usize::from_str(&s).ok()) {
+            match env::var("RAYON_RS_NUM_THREADS").ok().and_then(|s| usize::from_str(&s).ok()) {
```

Comment:
Sure, I can check for both. Should one be treated preferrentiably over the other?

Also, do you think it's a big concern? a) rayon is pre-1.0 :), b) how many people are blindly going to keep using the same env var after rayon hits 1.0 (which I heard is planned for soonish)? I think advertising the change in a prominent position should be fine?